### PR TITLE
Added gnome sort

### PR DESCRIPTION
**Pull request by ron-popov**

### CODE REVIEW

File: sort/gnome.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package sort
+
```
Comment (x899) [inline sort/gnome.go]: Please document the function. Check out other sorting algorithms for reference.

Comment (ron-popov) [reply sort/gnome.go]: Done

File: sort/gnome.go [hunk 2]
```diff
@@ -1,5 +1,9 @@
 package sort
 
+
+// The gnome sort is a sorting algorithm which is similar
```
Comment (x899) [inline sort/gnome.go]: Thanks. Another minor thing.
The comment always start with the function, variable, etc name.
So it should be "GnomeSort is a sorting algorithm...."

From [GoDoc Documentation](https://blog.golang.org/godoc-documenting-go-code):
> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.

Comment (ron-popov) [reply sort/gnome.go]: Is that ok ?

Comment (x899) [reply sort/gnome.go]: perfect.

**Comment by x899** [comment]
Thank you for your contribution. Here's your cookie 🍪 .
Feel free to add more algorithms.

**Pull request merged**